pcsx2/IPC.h

```diff
@@ -54,6 +54,11 @@ class SocketIPC : public pxThread
      */
 #define MAX_IPC_RETURN_SIZE 450000
 
+	/**
+     * Maximum number of socket errors in a row before the thread stops.
```

Comment:
Why multiple line code comments for 1 line?

Reply:
That was because they were originally doxygen-formatted comments, taken from the client implementation. 
Just fixed the styling to make it use // comments for 1 line comments.